app/assets/stylesheets/partials/_navigation.css.scss

```diff
@@ -0,0 +1,17 @@
+nav.navbar {
+  @include clearfix;
+  position: relative;
+  text-align: center;
+  clear: both;
+  width: 100%;
+  overflow: visible;
+  border-bottom: 1px solid $base-border-color;
+  line-height: 2;
+  cursor: pointer;
+  margin: em(30) 0;
+}
+nav li {
+  margin: .5em 0;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>`.5` should be written with a leading zero as `0.5`<br>Properties should be ordered display, margin